Command/AbstractSwiftMailerCommand.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SwiftmailerBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * @internal
+ */
+abstract class AbstractSwiftMailerCommand extends Command
+{
+    /**
+     * @var ContainerInterface|null
+     */
+    private $container;
+
+    /**
+     * @deprecated since SwiftMailerBundle 3.3
+     */
+    public function setContainer(ContainerInterface $container = null)
+    {
+        @trigger_error(sprintf('The "%s::setContainer()" method is deprecated since SwiftMailerBundle 3.3.', \get_class($this)), E_USER_DEPRECATED);
+
+        $this->container = $container;
+    }
```

Comment:
if we are ok with the parent class change in a patch, I can remove the deprecation here.

Reply:
We don't have the same strict policy as in Symfony core. So, no need to trigger a deprecation IMHO.